tests/components/yeelight/__init__.py

```diff
@@ -49,7 +49,9 @@
     "bg_flowing": "0",
     "bg_ct": "5000",
     "bg_bright": "80",
-    "bg_rgb": "16711680",
+    "bg_rgb": "65280",
+    "bg_hue": "200",
+    "bg_sat": "70",
```

Comment:
Are you sure this is correct?

Reply:
According to yeelight spec, https://www.yeelight.com/download/Yeelight_Inter-Operation_Spec.pdf, hue is in the range 0..359 and saturation in the range 0..100.

This is why I removed the conversion: https://github.com/home-assistant/core/pull/51973/files/0955c7269658633105b14cbbbc09162132523e76#diff-5fe9c3ca5816491ba88aab8cec171a10ab6f5e199923faadf128b09743daf861L579

Could you help to verify `hue`, `sat`, `bg_hue`, `bg_sat` are as documented and not in the ranges 0..65536, 0..255 as assumed by the old code?

Reply:
Oh, I see.

I changed the `bg_rgb` to make it different from `rgb` to be certain the integration picks the right property for the background light.

Reply:
Ah, great! Okay, let's get this merged then and handle fixes later on, if necessary :-)

Reply:
For me it looks good, and I think we should merge and tweak. I've also dig into upstream lib and there is also stated that hue is in range 0...359 and sat 0...100. https://gitlab.com/stavros/python-yeelight/-/blob/master/yeelight/main.py#L844